TeamsProject/main.py

```diff
@@ -0,0 +1,172 @@
+import selenium
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.chrome.options import Options
+import yaml
+import sys
+import time
+import pyautogui as magic
+from selenium.common.exceptions import NoSuchElementException
+from datetime import date
+from datetime import datetime
+
+
+sys.tracebacklimit = 0
+
+
+settings_path = "D:\Desktop\Python Documents\TeamsProject\settings.yaml"
+with open(settings_path) as f:
+    settings = yaml.load(f, Loader=yaml.FullLoader)
+
+
+logindetails = settings['logindetails']
+username = logindetails['username']
+password = logindetails['password']
+
+mon = settings['monday']
+tue = settings['tuesday']
+wed = settings['wednesday']
+thur = settings['thursday']
+fri = settings['friday']
+
+
+def initiation():
+    print("Class starting..")
+    print("Initiating Bot!")
+    print("Starting Teams")
+    global driver
+    options = webdriver.ChromeOptions()
+    options.add_experimental_option('excludeSwitches', ['enable-logging'])
+    driver = webdriver.Chrome(executable_path="C:\Program Files (x86)\chromedriver.exe", options=options)

```

Comment:
I am not sure if this repo has any instructors about having a single webdriver and where but this is also the same issue as the previous one.